### DISCUSSION TITLE
Standalone app

### DISCUSSION THREAD

**Discussion by robinsrk** [magit/magit#4760] [Ideas]
It would be great to have an standalone app with all the keybindings and features like other git clients.

**Comment by tarsius** [comment]
Magit runs inside Emacs; without Emacs, there's no Magit.

There's https://github.com/alphapapa/magit.sh to quickly launch a new Emacs instance with Magit ready to go.

A few tools exist that are somewhat similar to Magit or are even inspired, which work in a shell. See https://github.com/magit/magit/wiki/Other-Tools.
